scripts/feature_toggle_report_generator.py

```diff
@@ -314,6 +315,36 @@ def link_annotation_reports_to_idas(idas, annotation_report_files_path):
         idas[ida_name].annotation_report_path = annotation_file_path
 
 
+def generate_code_annotation_docs(idas, output_path):
+    """
+    For each IDA, copy its annotation report into the `output_path` directory
+    and generate code annotation docs. Finally, move this set of docs into
+    its own subdirectory
+    """
+    if os.path.isdir(output_path):
+        shutil.rmtree(output_path)
+    os.mkdir(output_path)
+    for ida_name, ida in idas.items():
+        if not ida.annotation_report_path:
+            continue
+        temp_annotation_report = os.path.join(
+            'reports', os.path.basename(ida.annotation_report_path)
+        )
+        shutil.copyfile(ida.annotation_report_path, temp_annotation_report)
+        annotation_config = AnnotationConfig('feature_annotations.yml', 0)
+        report_files = (io.open(temp_annotation_report, 'r'), )
```

Comment:
Probably a little cleaner to put this code in a `with io.open(temp_annotation_report, ...` context manager, to make sure the file is closed at the right time.